sdk/js/src/Request.ts

```diff
@@ -66,12 +66,16 @@ export const getConfigJson = async (
         const res = await getWithTimeout(url, requestConfig, 5000)
         return await res.json()
     } catch (e) {
+        const error = e as ResponseError
         logger.error(
             `Request to get config failed for url: ${url}, ` +
-                `response message: ${e}`,
+                `response message: ${error.message}`,
         )
+        if (error.status === 401 || error.status === 403) {
+            throw new Error(`Invalid SDK Key. Error details: ${error.message}`)
```

Comment:
you should use the `UserError` pattern we have in the Node SDK here rather than inspecting the error message contents.

Reply:
done! also updated the invalid sdk key errors to be `UserError`